tools/setup.py

```diff
@@ -0,0 +1,55 @@
+#!/bin/python3
+# Copyright 2023 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import date
+from glob import glob
+from setuptools import setup
+import os
```

Comment:
Nit: Python imports should be sorted/grouped into:
1. standard library imports (here, `datetime` and `os`),
2. third-party library imports (here, `glob` and `setuptools`),
3. project module imports (in this file, none).

Normally, pyink should fix this, but for some reason it merges third party and project imports. I need to investigate this.

Reply:
`glob` is in the standard library (as far as I'm aware), so I've grouped it with everything else and moved `setuptools` to a section below given it's the only third-party dependency. Thanks for the info! I should probably go through the Google python style guide more thoroughly at some point.